src/ansys/mapdl/core/mapdl.py

```diff
@@ -55,6 +55,7 @@
 _PERMITTED_ERRORS = [
     r"(\*\*\* ERROR \*\*\*).*(?:[\r\n]+.*)+highly distorted.",
     r"(\*\*\* ERROR \*\*\*).*[\r\n]+.*is turning inside out.",
+    r"(\*\*\* ERROR \*\*\*).*[\r\n]+.*The distributed memory parallel solution does not support KRYLOV method",
```

Comment:
```suggestion
    r"(\*\*\* ERROR \*\*\*).*[\r\n]+.*The distributed memory parallel solution does not support the Krylov method.",
```

Reply:
This cannot be done because this is the regex pattern we use to read MAPDL output, so it should be as it is.